interfaces/builtin/acrn.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Intel Corporation, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package builtin
+
+const acrnSummary = `allows access to the ACRN device`
+
+const acrnBaseDeclarationSlots = `
+  acrn:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const acrnConnectedPlugAppArmor = `
+# Description: Allow access to resources required by ACRN.
+#   allow offline CPU cores
+/sys/devices/system/cpu/cpu[0-9]*/online w,
+
+#   allow write access to ACRN Virtio and Hypervisor service Module
+/dev/acrn_vhm rw,
+
+#   allow configure tap network devices
+/dev/net/tun rw,
```

Comment:
this is already in network-control, I don't think we should include this here, if your application/snap needs to configure network devices it should use the `network-control` interface alongside this `acrn` interface.

Reply:
@anonymouse64 , I did try to connect the `network-control` interface, but it looks like there may some configurations blocking our accesses to the `/dev/acrn_vhm`, so I add the required `/dev/net/tun` permission to the snippet directly.

Reply:
Did you use snapd built from this branch to get the acrn and the network-control interfaces at the same time? perhaps @alfonsosanchezbeato can help you debug this? There shouldn't be anything in network-control that conflicts with this new acrn interface providing access to `acrn_vhm`.